types/lib/associations/has-many.d.ts

```diff
@@ -1,14 +1,15 @@
 import { DataType } from '../data-types';
 import {
-  CreateOptions,
+  CreateOptions, CreationAttributes,
```

Comment:
And put this on a new line as well?

Reply:
I wish the linter would handle that, maybe at some point